packages/core/src/components/forms/formGroup.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import * as classNames from "classnames";
+import * as PureRender from "pure-render-decorator";
+import * as React from "react";
+import * as Classes from "../../common/classes";
+import { IIntentProps, IProps } from "../../common/props";
+
+export interface IFormGroupProps extends IIntentProps, IProps {
+    /**
+     * Whether form group should appear as non-interactive.
+     * Remember that input elements must be disabled separately
+     */
+    disabled?: boolean;
+
+    /** Optional helper text. Will be wrapped in `.pt-form-helper-text` and displayed beneath `children`. */
+    helperText?: React.ReactNode;
+
+    /** Whether to render label and control on a single line. */
+    inline?: boolean;
+
+    /** Label of this form group */
```

Comment:
Missing period: `group.`

Reply:
nit: Might also phrase as `label for...`, if it's not too confusing that we also have a `labelFor` prop. :)